packages/contracts/src/dollar/libraries/LibDollarMintExcess.sol

```diff
@@ -156,11 +157,12 @@ library LibDollarMintExcess {
         );
 
         // deposit liquidity
-        uint256 res = IMetaPool(stableSwapMetaPoolAddress).add_liquidity(
-            [0, amount3CRVReceived],
-            0,
-            address(this) // stacking contract
-        );
+        uint256 res = ICurveStableSwapMetaNG(stableSwapMetaPoolAddress)
+            .add_liquidity(
```

Comment:
Perhaps its best to run the formatting with the same rules as before to minimize these sort of changes?

Reply:
I don't understand, this is formatted code by https://github.com/ubiquity/ubiquity-dollar/blob/9f882429a844ca14118079c255368446bbafa8de/package.json#L55

Reply:
+1, lets reformat